miniserv.pl

```diff
@@ -1704,7 +1704,7 @@ sub handle_request
 	}
 
 # Check for SSL authentication
-my $trust_ssl = $config{'trust_real_ip'} && !$config{'no_trust_ssl'};
+my $trust_ssl = $config{'trust_real_ip'} == 2;
```

Comment:
Why not just keep these two config variables internally, but show only one selector in the UI?

Reply:
There is no need for an additional variable because we now combine it under one radio element. Adding it would represent unnecessary complexity.

Reply:
Yes, but you have moved the complexity elsewhere!